pkg/logql/syntax/ast.go

```diff
@@ -229,6 +230,17 @@ func newLineFilterExpr(ty labels.MatchType, op, match string) *LineFilterExpr {
 	}
 }
 
+func newOrLineFilter(left, right *LineFilterExpr) *LineFilterExpr {
```

Comment:
Make a new OrLineFilter from left and right

Reply:
I vaguely remember that the ordering was reversed. At least for the `ContainsAllFilter`.